test/include/nntrainer_test_util.h

```diff
@@ -191,6 +191,20 @@ class ScopedIni {
     }                                             \
   } while (0)
 
+#define GEN_TEST_INPUT_DOUBLE(input, equation_i_j_k_l)        \
+  do {                                                        \
+    for (int i = 0; i < batch; ++i) {                         \
+      for (int j = 0; j < channel; ++j) {                     \
+        for (int k = 0; k < height; ++k) {                    \
+          for (int l = 0; l < width; ++l) {                   \
+            float val = (equation_i_j_k_l);                   \
```

Comment:
This is float. The caller in unittest_blas_kernel_cl.cpp is using double... is this intentional? (it's ok if it's intentional.